tests_react_native/example/app.tsx

```diff
@@ -237,6 +237,18 @@ function Root(): any {
               console.log(await Notifee.openPowerManagerSettings());
             }}
           />
+          <Button
+            title={`check can create timestamp trigger`}
+            onPress={async (): Promise<void> => {
+              console.log(await Notifee.canCreateTimeStampTrigger());
```

Comment:
helps keep things straight in the log - noticed while testing it was just true/false, this makes it easier to read

```suggestion
              console.log('Notifee.canCreateTimeStampTrigger: ' + await Notifee.canCreateTimeStampTrigger());
```